src/candela/components/TrackerDash/ErrorBulletWidget.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'underscore';
+import d3 from 'd3';
+import nv from 'nvd3';
+import $ from 'jquery';
+
+import { computeColor } from './utility.js';
+import VisComponent from '../../VisComponent';
+
+class ErrorBulletWidget extends VisComponent {
```

Comment:
I'm ok either way - this is still an "internal" component of TrackerDash, so if/when we promote this to a top-level component, we'd certainly want to change the name. I left the name as-is since I didn't vet any of the design for any "internal" components for general-purposeness; essentially, I'm still thinking of these as specifically designed for TrackerDash.

Let me know if you think we should change the name now, and do a heavier code-review type of step later.